backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VirtualQueueController.java

```diff
@@ -52,4 +54,9 @@ public GetTokensResponse getTokens(@PathVariable UUID shopId) {
     public GetShopsByShopOwnerResponse getShopsByShopOwner(@PathVariable String shopOwnerId) {
         return virtualQueueService.getShopsByShopOwner(shopOwnerId);
     }
+
+    @DeleteMapping("/token/{tokenId}")
+    public DeleteTokenResponse deleteToken(@PathVariable UUID tokenId, @RequestBody Boolean isLoggedin){
```

Comment:
According to camelCase, we should name the variable as "isLoggedIn" I think (instead of "isLoggedin").

Reply:
done!